$MirOS: ports/security/gnupg/patches/patch-mpi_config_links,v 1.3 2006/09/09 14:12:01 tg Exp $
--- mpi/config.links.orig	Tue Mar 25 10:15:24 2008
+++ mpi/config.links	Tue Apr  1 15:31:24 2008
@@ -42,6 +42,7 @@ case "${host}" in
     i[3467]86*-*-freebsd*-elf  | \
     i[3467]86*-*-freebsd[3-9]* | \
     i[3467]86*-*-freebsdelf*   | \
+    i[34567]86*-*-mirbsd*      | \
     i[3467]86*-*-netbsd*       | \
     i[3467]86*-*-k*bsd*)
        echo '#define ELF_SYNTAX' >>./mpi/asm-syntax.h
@@ -52,6 +53,7 @@ case "${host}" in
     i586*-*-freebsd*-elf   | \
     i586*-*-freebsd[3-9]*  | \
     i586*-*-freebsdelf*    | \
+    i[34567]86*-*-midnightbsd* | \
     i586*-*-netbsd*	   | \
     i586*-*-k*bsd*	   | \
     pentium-*-netbsd*	   | \
@@ -136,7 +138,7 @@ case "${host}" in
 	path="sparc32v8 sparc32"
 	mpi_extra_modules="udiv"
         ;;
-    sparc64-*-netbsd* | sparc64-*-freebsd* | sparc64-*-openbsd*)
+    sparc64-*-netbsd* | sparc64-*-freebsd* | sparc64-*-openbsd* | sparc64-*-midnightbsd*)
        # There are no sparc64 assembler modules that work on the
        # *BSDs, so use the generic C functions.
        echo '/* No working assembler modules available */' >>./mpi/asm-syntax.h
@@ -175,6 +177,10 @@ case "${host}" in
 
     # Motorola 68k configurations.  Let m68k mean 68020-68040.
     # mc68000 or mc68060 configurations need to be specified explicitly
+    m68k*-*-openbsd*)
+       echo '/* No working assembler modules available */' >>./mpi/asm-syntax.h
+       path=""
+       ;;
     m680[234]0*-*-linuxaout* | \
     m68k*-*-linuxaout*)
 	echo '#define MIT_SYNTAX'           >>./mpi/asm-syntax.h
